slidesimport/slidesimport.py

```diff
@@ -127,5 +129,35 @@ def run(rawArgs=None):
         mediaFilePath = getMediaPath(collectionMediaPath, prefix, slideNum)
         mediaFileName = getMediaName(prefix, slideNum)
 
-        outputDeckFile.write('"{0}"; <img src="{1}" />\n'.format(cgi.escape(qs), mediaFileName))
+        fallback_flag = False
+
+        # If the "question without slide" dict entry is not empty, write that
+        if notesQWS[slideNum] != '':
+            outputDeckFile.write('"{0}"; '.format(cgi.escape(notesQWS[slideNum])))
+        # Otherwise try "question followed by slide"
+        elif notesQFBS[slideNum] != '':
+            outputDeckFile.write('<div>{0}</div><img src="{1}" />; '.format(cgi.escape(notesQFBS[slideNum]), mediaFileName))
+        # Otherwise try "slide followed by question"
+        elif notesSFBQ[slideNum] != '':
+            outputDeckFile.write('<img src="{0}" /><div>{1}</div>; '.format(mediaFileName, cgi.escape(notesSFBQ[slideNum])))
+        else:
+            fallback_flag = True
+
+        # If the "answer without slide" dict entry is not empty, write that
+        if notesAWS[slideNum] != '':
+            outputDeckFile.write('"{0}"\n'.format(cgi.escape(notesAWS[slideNum])))
+        # Otherwise try "answer followed by slide"
+        elif notesAFBS[slideNum] != '':
+            outputDeckFile.write('<div>{0}</div><img src="{1}" />\n'.format(cgi.escape(notesAFBS[slideNum]), mediaFileName))
+        # Otherwise try "slide followed by answer"
+        elif notesSFBA[slideNum] != '':
+            outputDeckFile.write('<img src="{0}" /><div>{1}</div>\n'.format(mediaFileName, cgi.escape(notesSFBA[slideNum])))
+        else:
+            fallback_flag = True
```

Comment:
Out of curiosity, there are no delimiters (_i.e._ `;`) written to the `outputDeckFile` in the `* answer *` section. I am not sure how Anki behaves if delimiters are absent from some rows in an Anki file. Does it work and produce the expected behavior?